boards/nucleo-f091rc/Makefile.features

```diff
@@ -1,5 +1,7 @@
 # Put defined MCU peripherals here (in alphabetical order)
 FEATURES_PROVIDED += periph_adc
+FEATURES_PROVIDED += periph_dma
+FEATURES_PROVIDED += periph_gpio periph_gpio_irq
```

Comment:
This line is not needed anymore (see #10637)

Reply:
fixed